apps/typeform/lambda/app.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const path = require('path');
+const express = require('express');
+const fetch = require('node-fetch');
+const handleForms = require('./forms-handler');
+const handleWorkspaces = require('./workspaces-handler');
+const fetchAccessToken = require('./fetch-access-token');
```

Comment:
Nit: Why not use `import` syntax?

Reply:
I was following the lambda structure from `ai-image-tagging` which is just plain node and no babelconfig or any transpilation step. Also not happy with that configuration there

Reply:
Typescript should transpile this as well

Reply:
I just want to remind you that the only env we're targeting is modern node, either local or on AWS. In general you don't need to transpile to get all the new syntax/methods in node since node 8

In cases we don't use TypeScript (and IMO for projects of 200 LOC benefits of types are negligible) I'd recommend not to introduce extra build steps just to be able to `import` (and again benefits of imports in node-world are negligible, you don't need to bundle/tree shake server code).

BTW it's a little bit funny to see how people treat transpilation as a default and miss it when it's not in place ;)

Reply:
That's true, I didn't realize this wasn't in typescript 😆. However, that being said I disagree that typescript is only beneficial on large projects. Types help describe many aspects of the code and can be useful.

As for import syntax, it isn't available to the node envs we use since it would require node >= 13 without any explicit flag setting like in node 12.

Reply:
Even in node 13 ESM support is 1 - experimental hence I wouldn't use it.

When it comes to TypeScript, nothing against using it in projects of any size but for extra-small ones (especially these fitting two screens with trivial logic) in many cases I experienced that setup and maintenance (more tooling/deps to keep up-to-date) time is greater than time saved by having types. But ultimately it's your call ;)

Reply:
For me personally it was just confusing what to use now for the lambda functions. Which structure / project setup is the default or common way on how you implement lambda functions. But my opinion is always to keep it as simple as possible